src/figurePage.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
If URL is someting like this: `figures/shower_guy` this page doesn't really work